libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -626,11 +624,8 @@ bool WaveTrack::LinkConsistencyFix(bool doFix)
       }
       else if (doFix) {
          // non-error upgrades happen here
-         auto newLinkType =
-            AreAligned(SortedClipArray(), next->SortedClipArray())
-            ? LinkType::Aligned : LinkType::Group;
-         if (newLinkType != linkType)
-            SetLinkType(newLinkType);
+         if (!AreAligned(SortedClipArray(), next->SortedClipArray()))
+            SetLinkType(LinkType::None);
```

Comment:
Again, spelling in the commit comment.

Here, I think you miss the case that an old enough legacy file, with well aligned tracks, still has the legacy value of 1 which must be upgraded to `Aligned`.  Which did happen before this change.

Reply:
Yes, thanks